Algorithms/Search/exponential_search.cs

```diff
@@ -0,0 +1,65 @@
+
+using System; 
+class Gfg { 
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6263329)